examples/dial/dial.ino

```diff
@@ -0,0 +1,37 @@
+#include <A6lib.h>
+
+// Instantiate the library with TxPin, RxPin.
+A6lib A6l(7, 8);
+
+void setup() {
+    Serial.begin(115200);
+
+    delay(1000);
+
+    // Power-cycle the module to reset it.
+ //   A6l.powerCycle(D0);
```

Comment:
Well actually, it is because of the difference between your connection settings and mine.  I forget to turn it to what was it initially.